### ISSUE TITLE
Suggestion: automation for Extra Planets ores (Mercury, Red Gem, etc.)

### ISSUE THREAD

**Issue by AdityaGupta1** [Freebi85/Dungeons-Dragons-Space-Shuttles#695]
I think some infinite source of these ores, which are currently obtainable only by going to their respective planets, would be really nice for automation. I was thinking maybe new mystical agriculture seeds so they would be gated behind finding the material in the first place, or maybe add them to a high-tier void miner. Another way could be through Alchemistry - before 7.0, I used the mercury element to automate mercury ingots, but that recipe has been removed.

**Comment by Freebi85** [comment]
Hi :)
I'ts not possible to add new custom seeds in 1.12. This needs to be suggestet to the mod author of Mystical but im sure he's done with the 1.12. version.
Alchemistry isn't an option as well because it's not the goal of getting these ores before you can reach this planets. That's why it was removed.
Maybe the last tier of Void ore miners which requires a massive rebalancing of all the miners to fit into planet progression up to Tier10 planets. But I'm in a pre final release so not sure if this will happen for this pack,
Basically, planets are there to be explored and special planetary ores are not used in masses as for example for Avaritia.
Maybe i will find another solution as soon as i can invest more time in the project.
